lib/tool_belt/cherry_picker.rb

```diff
@@ -27,7 +27,7 @@ def find_cherry_picks(project, release, repo_names)
         commits = issue['changesets']
 
         commits.each do |commit|
-          if !commit['comments'].start_with?('Merge pull request') && !@release_environment.commit_in_release_branch?(repo_names, commit['comments'])
+          if commit['comments'].downcase.start_with?('fixes') && !@release_environment.commit_in_release_branch?(repo_names, commit['comments'])
```

Comment:
@ehelms So if I create a PR with a Refs because it doesn't actually resolve all of the redmine issue, that PR is merged, then I finish with a Fixes and that PR is merged. Now the redmine issue is closed; I would need to know both commit hashes for bring the necessary changes downstream; right?